moml/models/mgnn/evaluation/metrics.py

```
@@ -1,29 +1,54 @@
     mean_absolute_error,
-    r2_score,
+    mean_squared_error,
     precision_score,
+    r2_score,
     recall_score,
-    f1_score,
     roc_auc_score,
-    accuracy_score,
-    confusion_matrix,
 )
+from sklearn.utils.multiclass import type_of_target
```

Ruff (0.12.2)

`sklearn.utils.multiclass.type_of_target` imported but unused

Remove unused import: `sklearn.utils.multiclass.type_of_target`

(F401)